private/bufpkg/bufcheck/bufbreaking/bufbreaking_test.go

```diff
@@ -173,29 +173,29 @@ func TestRunBreakingFieldSameJSONName(t *testing.T) {
 		bufanalysistesting.NewFileAnnotation(t, "1.proto", 7, 18, 7, 35, "FIELD_SAME_JSON_NAME"),
 		bufanalysistesting.NewFileAnnotation(t, "1.proto", 8, 20, 8, 37, "FIELD_SAME_JSON_NAME"),
 		bufanalysistesting.NewFileAnnotation(t, "1.proto", 9, 3, 9, 27, "FIELD_SAME_JSON_NAME"),
-		bufanalysistesting.NewFileAnnotation(t, "1.proto", 10, 28, 10, 45, "FIELD_SAME_JSON_NAME"),
-		bufanalysistesting.NewFileAnnotation(t, "1.proto", 11, 27, 11, 44, "FIELD_SAME_JSON_NAME"),
+		bufanalysistesting.NewFileAnnotation(t, "1.proto", 10, 28, 10, 46, "FIELD_SAME_JSON_NAME"),
```

Comment:
I'm concerned that any of these changed. If anything, I'd expect that part of our test for if this protoparse-to-protocompile conversion worked would be that this file would have no diff. Can you explain here a bit?

Reply:
I had to change JSON names for the files to actually compile. I added dumb number suffixes, which made some of the spans grow by one.